### PR TITLE
docs(schema): add deprecation policy

### PR DESCRIPTION
**Pull request by markborkum**

### CODE REVIEW

**Review by macintoshpie — APPROVED** [review]
Looks great 👍

**Comment by corymosiman12** [comment]
Putting it in the schema seems fine but can we also add it to developer docs?

**Review by corymosiman12 — CHANGES_REQUESTED** [review]
Also add to developer docs

**Comment by markborkum** [comment]
> Putting it in the schema seems fine but can we also add it to developer docs?

I'd prefer not. If the deprecation policy is an intrinsic part of schema file, then it is exchanged with the schema file and it travels with the schema file, wherever it goes. Having an alternate representation of the deprecation policy in the `docs/` folder is disadvantageous because it would force us to either maintain a duplicate the same content or to maintain a URL, which must be resolvable, within the schema file to a "raw" file (e.g., in the GitHub repository).

**Comment by corymosiman12** [comment]
Sure I mean maybe we just make a note in the developer docs that the deprecation policy is defined by the schema doc.  My point is most people look to documentation on Github.  It wouldn't be intuitive to me that the deprecation policy is defined in the Schema itself.

**Review by corymosiman12 — APPROVED** [review]
🌭

File: docs/developer_resources.md [hunk 1]
```diff
@@ -44,3 +44,7 @@ Pull request examples:
 
 - `#1234 feat(schema): add MyElement to ParentElement`
 - `proposal: add proposal for MyElement`
+
+## Deprecation Policy
+
+Details of the deprecation policy are included in the BuildingSync XML schema file.
```
Comment (nllong) [inline docs/developer_resources.md:50]: 👍

**Review by nllong — APPROVED** [review]
looks great.

**Pull request merged**